src/main/scala/org/broadinstitute/dsde/firecloud/service/WorkspaceService.scala

```diff
@@ -6,7 +6,7 @@ import javax.ws.rs.Path
 
 import akka.actor.{Actor, Props}
 import com.wordnik.swagger.annotations._
-
+import org.broadinstitute.dsde.firecloud.utils.TSVParser
```

Comment:
Nit: This should move down per the [import ordering conventions](https://broadinstitute.atlassian.net/wiki/display/DSDE/Code+Style+Conventions)